pkg/cache/redis_test.go

```diff
@@ -0,0 +1,22 @@
+// Copyright © 2020 by PACE Telematics GmbH. All rights reserved.
+// Created at 2020/08/12 by Marius Neugebauer
+
+package cache_test
+
+import (
+	"testing"
+
+	"github.com/pace/bricks/backend/redis"
+	"github.com/pace/bricks/pkg/cache"
+	"github.com/pace/bricks/pkg/cache/testsuite"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestIntegrationRedis(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
+	suite.Run(t, &testsuite.StringsTestSuite{
+		Cache: cache.InRedis(redis.Client(), "test:cache:"),
```

Comment:
I think you should flush all redis keys with the given prefix before and after the test to make the test more robust against multiple re runs and parallel execution.

Reply:
How do I do that?

Reply:
I found this: https://stackoverflow.com/questions/4006324/how-to-atomically-delete-keys-matching-a-pattern-using-redis

Reply:
I clean up every key in the tests, so I think this should be fine.